### ISSUE TITLE
About train_loss2

### ISSUE THREAD

**Issue by eyuansu62** [lorenmt/maxl#2]
In the meta-training step:
# update theta_2 with primary loss + entropy loss
        (torch.mean(train_loss1) + 0.2*torch.mean(train_loss3)).backward()
        gen_optimizer.step()
why its not train_loss2?

**Comment by lorenmt** [comment]
The auxiliary task is designed solely to help the learning of the primary task. Thus, we hope the theta_2 is updated only based on the loss for the primary task, which is the reason we do no include the auxiliary task loss in the label generation network update.

**Comment by eyuansu62** [comment]
Thanks a lot!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Shikun Liu"<notifications@github.com&gt;;
发送时间:&nbsp;2020年6月19日(星期五) 凌晨2:19
收件人:&nbsp;"lorenmt/maxl"<maxl@noreply.github.com&gt;;
抄送:&nbsp;"末冬末秋"<772468951@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [lorenmt/maxl] About train_loss2 (#2)





 
 The auxiliary task is designed solely to help the learning of the primary task. Thus, we hope the theta_2 is updated only based on the loss for the primary task, which is the reason we do no include the auxiliary task loss in the label generation network update.
